### PR TITLE
Added sql dtc option

### PR DESCRIPTION
**Pull request by johnsimons**
image

### CODE REVIEW

**Comment by johnsimons** [comment]
@mikeminutillo this is it

File: src/ServiceControlInstaller.Engine/Instances/Transports.cs [hunk 1]
```diff
@@ -17,37 +17,44 @@ public static TransportInfo FindByName(string name)
             {
                 Name = "AzureServiceBus",
                 TypeName = "NServiceBus.AzureServiceBusTransport, NServiceBus.Azure.Transports.WindowsAzureServiceBus",
-                MatchOn = "NServiceBus.AzureServiceBus",
+                MatchOn = "NServiceBus.AzureServiceBusTransport",
```
Comment (gbiellem) [inline src/ServiceControlInstaller.Engine/Instances/Transports.cs]: @johnsimons Changing the match value will break upgrades from older ServiceControl that are still on NSB v4.  The V4 transports didn't have "Transport" at the end of the string.

File: src/ServiceControlInstaller.Engine/Instances/Transports.cs [hunk 2]
```diff
@@ -17,37 +17,44 @@ public static TransportInfo FindByName(string name)
...
                 Name = "SQLServer",
                 TypeName = "NServiceBus.SqlServerTransport, NServiceBus.Transports.SQLServer",
-                MatchOn = "NServiceBus.SqlServer",
+                MatchOn = "NServiceBus.SqlServerTransport",
+                SampleConnectionString = @"Data Source=<SQLInstance>;Initial Catalog=nservicebus;Integrated Security=True"
+            },
+            new TransportInfo
+            {
+                Name = "SQLServer with DTC",
+                TypeName = "NServiceBus.SqlServerWithDTCTransport, NServiceBus.Transports.SqlServerWithDTC",
```
Comment (gbiellem) [inline src/ServiceControlInstaller.Engine/Instances/Transports.cs]: @johnsimons 

The TypeName is what ends up in the app config in ""ServiceControl/TransportType" so it that what you intents here?

Comment (johnsimons) [inline src/ServiceControlInstaller.Engine/Instances/Transports.cs]: yes

File: src/ServiceControlInstaller.Engine/Instances/Transports.cs (hunk 1 above)
Comment (johnsimons) [inline src/ServiceControlInstaller.Engine/Instances/Transports.cs]: In that case I may need to rename my custom transport to not conflict

File: src/ServiceControlInstaller.Engine/Instances/Transports.cs [hunk 3]
```diff
@@ -17,30 +17,30 @@ public static TransportInfo FindByName(string name)
             {
                 Name = "AzureServiceBus",
                 TypeName = "NServiceBus.AzureServiceBusTransport, NServiceBus.Azure.Transports.WindowsAzureServiceBus",
-                MatchOn = "NServiceBus.AzureServiceBusTransport",
+                MatchOn = "NServiceBus.AzureServiceBus,",
```
Comment (gbiellem) [inline src/ServiceControlInstaller.Engine/Instances/Transports.cs]: @johnsimons I think you made a typo in all the MatchOn values you put back - each one has a comma at the end

**Comment by johnsimons** [comment]
Replaced by https://github.com/Particular/ServiceControl/pull/757

**Pull request closed without merging**